.github/workflows/cicd.yml

```diff
@@ -152,6 +152,20 @@ jobs:
       code_coverage: false
       gather_dumps: true
 
+  # Run the verifier fuzzer.
+  verifier_fuzzer:
+    needs: libfuzzer
+    # Always run this job.
+    if: github.event_name == 'schedule' || github.event_name == 'pull_request'
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      name: verifier_fuzzer
+      test_command: verifier_fuzzer.exe verifier_corpus -use_value_profile=1 -max_total_time=1800
```

Comment:
Once the #1164 is merged add "-artifact_prefix=Artifacts\" to the test_command so that artifacts are uploaded.